src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -139,7 +141,7 @@ export const PhrasalTemplatePhrase = ({
               onClick={handlePreviewPhrase}
               sx={{ gap: 0.8, width: 'max-content' }}
               variant="text"
-              disabled={formState.isDirty || formState.isSubmitting || !formState.isValid}
+              disabled={formState.isSubmitting || !formState.isValid}
```

Comment:
This button's disabled state is not obvious. The only difference is it lacks a hover effect, and the click action bubbles up to its container. When I first tested your work, I kept thinking my app hadn't refreshed from the right git branch, because clicking it was just toggling the card's collapse state. It wasn't until I realized that the form state was invalid (which has no visual indicator until you've tried to click Save & Publish at least once) that the button was simply disabled.

![CleanShot 2024-10-25 at 10 00 07](https://github.com/user-attachments/assets/c6f3c236-d2c9-493d-96fd-2c55be7a5b33)

So I guess at a minimum, this button variant should have [proper disabled styling](https://www.figma.com/design/LXl9BAJSwtgWuzkxqOqvX8/ML1-Design-System?node-id=57868-142695&m=dev).

Besides the styling issue, I'm also not sure about the UX of clicking a disabled button resulting in its ancestor's click handler to execute. I think it would feel more natural if nothing happened when I clicked it when disabled, and the card expand toggle handler was limited to its surrounding area instead, but I'm not strongly attached to that.

Reply:
I pushed up some changes that should address this.

First, I fixed the appearance of the disabled state for the button's `text` variant — the theme configuration included some selector specificity conflicts that prevented the correct text color from being applied.

Since this is a global change, I clicked around other locations where we were using the `text` variant with a (conditionally) disabled state (eg, The "Feedback" & "Download Latest Report" buttons in `/dashboard/{appletId}/participants/{subjectId}/activities/{activityId}/{summary | responses}`) — there's not that many and there doesn't seem to be any adverse effect elsewhere 😀.

So that solves one problem. The second issue — the nested interactive elements behaviour — is almost identical in cause to the one I nagged you about in your [PDP pull request](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1943). 😭

My solution here is a bit of a kludge. I was hoping to do something similar to yours in that PR, but we're using slightly different components (`Accordion` vs `Collapse`) and there's a slightly different desired behaviour (we want a `disabled` button to continue to prevent clicks on the parent element). 

In the end I just added an additional `onClick` with a comment explaining the purpose on the wrapper node around both this and the "Delete Phrase" button, that simply calls `stopPropagation`. This works to stand-in as a handler for the button when it's in a disabled state, and avoids misclicks if you happen to press in the ambiguous boundary between the two buttons.